extension/ts/services/emitShowToast.ts

```diff
@@ -1,7 +0,0 @@
-import {SHOW_TOAST_EVENT} from "../constants";
```

Comment:
This file no longer needed as there is just js file loaded, so we call `showToast` directly. Even works with the (now removed) toast counter